services/brig/brig.integration.yaml

```diff
@@ -12,6 +12,10 @@ elasticsearch:
   url: http://127.0.0.1:9200
   index: directory_test
 
+cargohold:
+  host: 127.0.0.1
+  port: 8084
```

Comment:
Could you update the diagram from https://github.com/wireapp/wire-server/tree/develop/doc/arch using e.g. https://www.draw.io/ to reflect that brig needs routing to cargohold?

Reply:
👍

Reply:
https://github.com/wireapp/wire-server/pull/296/commits/afac44a67437d1d9e089ec6bf8ee84cbea386a3d